python/ray/dashboard/tests/conftest.py

```diff
@@ -10,13 +9,8 @@
 @pytest.fixture
 def enable_test_module():
     os.environ["RAY_DASHBOARD_MODULE_TEST"] = "true"
-    import ray.dashboard.tests
```

Comment:
Do you know why we need this before and doesn't need it for now?

Reply:
I'm not sure why we needed this before, but all tests pass even without it now.

This is only one of the causes of the flaky tests because the tests still retry 2 times before passing. I am finding the second cause now.